lib/cli-clone-report.ts

```diff
@@ -0,0 +1,60 @@
+import * as powerbi from 'powerbi-api';
+import * as msrest from 'ms-rest';
+import {Cli as cli} from './cli';
+import {Config as config} from './config';
+import * as program from 'commander';
+
+export default function CliCloneReprot() {
+    let pkg = require('../package.json');
+
+    program.version(pkg.version)
+        .option('-c, --collection <collection>', 'The Power BI workspace collection')
+        .option('-w, --workspace <workspace>', 'The Power BI workspace')
+        .option('-r, --report <reportKey>', 'The report to clone')
+        .option('-k, --accessKey <accessKey>', 'The Power BI workspace collection access key')
+        .option('-n, --newName <newName>', 'The new report name')
+        .option('-t, --tworkspace [targetWorkspace]', 'New target workspace to save report in. default is original report workspace')
```

Comment:
When moving between workspaces, the dataset has to reside within the targeted workspace